src/main/webapp/blobstoreFunctions.js

```diff
@@ -0,0 +1,54 @@
+/** Loads photo */
+function loadScripts() {
+  fetchBlobstoreUrl();
+  populatePosts();
+}
+
+
+/** Loads upload image URL and attaches it to the form */
+function fetchBlobstoreUrl() {
+  fetch('/blobstore-upload-url') 
+      .then((response) => {
+        return response.text();
+      })
+      .then((imageUploadUrl) => {
+        const messageForm = document.getElementById('plant-pic-form');
+        messageForm.action = imageUploadUrl;
```

Comment:
I think you could probably do document.getElementById('plant-pic-form).action = imageUploadUrl instead of creaiting a variable here, unless are you using this variable later for some reason?

Reply:
You're right, that works I'll change that